config/travis/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [ "$TRAVIS_BRANCH" != "master" ] ; then
+  echo "This commit was made against $TRAVIS_BRANCH and not master. No build for you!"
```

Comment:
Instead of `No build for you!`, should we give more build context and say something like `Not generating updated docs on gh-pages branch.`? same for line 13

Reply:
FAIR